### ISSUE TITLE
Crash on widget/filter creation (ClassNotFoundException: FilterScriptFragment).

### ISSUE THREAD

**Issue by kevjava** [mpcjanssen/simpletask-android#476]
Upon attempting to create a widget on the home screen, the application crashes.  The following stack trace is printed to the log:

```
07-11 16:47:37.208 9085-9085/? E/AndroidRuntime: FATAL EXCEPTION: main
                                                 Process: nl.mpcjanssen.todotxtholo, PID: 9085
                                                 java.lang.NoClassDefFoundError: Failed resolution of: Lnl/mpcjanssen/simpletask/FilterScriptFragment;
                                                     at nl.mpcjanssen.simpletask.FilterActivity.onCreate(FilterActivity.kt:141)
                                                     at android.app.Activity.performCreate(Activity.java:6658)
                                                     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
                                                     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2584)
                                                     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2692)
                                                     at android.app.ActivityThread.-wrap12(ActivityThread.java)
                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1445)
                                                     at android.os.Handler.dispatchMessage(Handler.java:102)
                                                     at android.os.Looper.loop(Looper.java:154)
                                                     at android.app.ActivityThread.main(ActivityThread.java:6044)
                                                     at java.lang.reflect.Method.invoke(Native Method)
                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
                                                  Caused by: java.lang.ClassNotFoundException: Didn't find class "nl.mpcjanssen.simpletask.FilterScriptFragment" on path: DexPathList[[zip file "/data/app/nl.mpcjanssen.todotxtholo-1/base.apk"],nativeLibraryDirectories=[/data/app/nl.mpcjanssen.todotxtholo-1/lib/arm64, /system/lib64, /vendor/lib64]]
                                                     at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
                                                     at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
                                                     at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
                                                     at nl.mpcjanssen.simpletask.FilterActivity.onCreate(FilterActivity.kt:141) 
                                                     at android.app.Activity.performCreate(Activity.java:6658) 
                                                     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118) 
                                                     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2584) 
                                                     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2692) 
                                                     at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1445) 
                                                     at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                     at android.os.Looper.loop(Looper.java:154) 
                                                     at android.app.ActivityThread.main(ActivityThread.java:6044) 
                                                     at java.lang.reflect.Method.invoke(Native Method) 
                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865) 
                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755) 
```

My hardware is an LG Nexus 5X running the updated N Developer Preview (build NPD56N).  Please let me know if you need more information or have experiments you'd like me to try.

**Comment by mpcjanssen** [comment]
Should be fixed in 8.0.2 can you confirm?

**Comment by kevjava** [comment]
I can confirm it.  Widget creation works now.  Thanks, Mark - that was fast.
